classes/observers/traits/Publicationable.inc.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @file classes/observers/events/BasePublicationEvent.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class BasePublicationEvent
+ * @ingroup core
+ *
+ * @brief Base event for publications being published or unpublished
+ */
+
+namespace PKP\observers\traits;
+
+use PKP\publication\PKPPublication;
+use PKP\submission\PKPSubmission;
+
+trait Publicationable
```

Comment:
Should this be `Publishable` rather than `Publicationable`?

Reply:
Sorry for my limited English. That was in my example, meaning trait which suits for different actions with publication rather than only publish/unpublish. But if it's only for these 2 events, `Publishable` is definitely better.